example/Mock.elm

```diff
@@ -60,7 +63,20 @@ bar a b =
     ... |> List.map toString
     ... |> join ""
     "321"
+
 -}
 rev : List a -> List a
 rev =
     List.reverse
+
+
+{-| some inline comments
+
+    >>> -- comment
+    >>> quux 10 32 -- another
+    42 --result
```

Comment:
I'm curious how this actually looks when generated as docs. Do you have an example?